run_scripts/1_s03_kakuma_coda_get.sh

```diff
@@ -20,7 +20,8 @@ DATASETS=(
     "kakuma_s03e02"
     "kakuma_s03e03"
     "kakuma_s03e04"
-    "kakuma_s03e05"
+    "kakuma_s03e05",
```

Comment:
I think this needs to not have a comma otherwise bash might do something weird

Reply:
Thanks for catching this. updated.